rollup-interface/README.md

```diff
@@ -20,6 +20,8 @@ To learn how the Sovereign SDK works, see the ![Sovereign SDK Overview](./specs/
 The Sovereign SDK is a research prototype. It has not been audited and should not be used in production under any
 circumstances.
 
+* Currently acceptable hashes for Spec should fit into 32 bytes.
```

Comment:
This doesn't seem to fit into its context. Was it meant to go here?

Reply:
What would be better place for it? I just wanted to reflect our decision to stick with 32 bytes hashes

Reply:
Where should it be placed?

Reply:
Moved it